### PR TITLE
Portfolio add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by saqibjavaiddev**
Portfolio: add a contact form and finish the mobile version. I have created a contact form section and for this I have:

- Created a contact form using HTML
- Styled the contact form using CSS and mainly used **Flexbox** Layout.
- Used formspree for form submission

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,6 +213,20 @@ <h2>LET’S CONNECT</h2>
...
         </section>
+        <section class="contact-section" id="contact">
+          <div class="txt">
+          <h2>Contact me</h2>
+          <p>If you have an application you are interested in developing, 
+            a feature that you need built or a project that needs coding. 
+            I’d love to help with it </p>
+            </div>
+        <form action="https://formspree.io/f/mlezzawk" method="post">
+          <input type="text" id="fullname" name="fullname" placeholder="Your name.." required maxlength="20">
```
Comment (GzimAsani) [inline index.html:224]: - [ ] The max length of the first name should be 30

File: index.html [hunk 2]
```diff
@@ -213,6 +213,20 @@ <h2>LET’S CONNECT</h2>
...
+          <input type="email" id="emailaddress" name="emailaddress" placeholder="Your Email Address" required>
+          <textarea name="content" id="textarea" cols="30" rows="10" required maxlength="500" placeholder="Write your message here"></textarea>
+      <input type="submit" id="submitBtn" value="Get In Touch" />
```
Comment (GzimAsani) [inline index.html:227]: - [ ] Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the Figma template.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @saqibjavaiddev   
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job so far :clap: 
 
 ## Required Changes 

- [ ] _Check the comments below :arrow_double_down:_


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (saqibjavaiddev) [reply index.html:227]: Hover effect added.

**Review by Lordkaito — APPROVED** [review]
Hi @saqibjavaiddev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**